Frontend/nix/bundle.nix

```diff
@@ -0,0 +1,55 @@
+{ ... }:
+{
+  imports = [
+    ../node
+  ];
```

Comment:
You don't need this because it is already imported at top-level.

Reply:
I like the fact that it documents at the use site where the nodeDependencies/nodejs version are coming from. I can remove it but was fully aware that it was redundant.